src/Workspaces/Core/Desktop/Workspace/Host/Mef/MefV1HostServices.cs

```diff
@@ -17,6 +17,10 @@ namespace Microsoft.CodeAnalysis.Host.Mef
     /// </summary>
     public class MefV1HostServices : HostServices, IMefHostExportProvider
     {
+        internal delegate MefV1HostServices CreationHook(IEnumerable<Assembly> assemblies);
+
+        private static CreationHook s_CreationHook;
```

Comment:
Would like a comment that this is test-only. I can imagine people doing really terrible things with this thinking this is a "feature".

Reply:
➡️ Added comments in 37a1d2c